[c10d] Change watchdog inner loop function name to make it more accurate

fduwjj opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #115404

Function `workCleanupLoop` does not affect all things we did in watchdog thread, so proposing a new name here to reflect what we are actually doing in the watchdog thread.

cc @mrshenli @pritamdamania87 @zhaojuanmao @satgera @rohan-varma @gqchen @aazzolini @osalpekar @jiayisuse @H-Huang @kwen2501 @awgu @penguinwu @fegin @XilunWu @wanchaol @wz337 @tianyu-l @wconstab @yf225 @kiukchung @lucasllc

fduwjj committed:
[c10d] Change watchdog inner loop function name to make it more accurate

[ghstack-poisoned]

kwen2501 reviewed (approved):
Thanks for cleaning the naming!
nit: can we remove the word "handler"? `watchdogLoop` should read just fine :)

fduwjj committed:
Update on "[c10d] Change watchdog inner loop function name to make it more accurate"


Function `workCleanupLoop` does not affect all things we did in watchdog thread, so proposing a new name here to reflect what we are actually doing in the watchdog thread.

cc mrshenli pritamdamania87 zhaojuanmao satgera rohan-varma gqchen aazzolini osalpekar jiayisuse H-Huang kwen2501 awgu penguinwu fegin XilunWu wanchaol wz337 tianyu-l wconstab yf225 kiukchung lucasllc

[ghstack-poisoned]

fduwjj commented:
@pytorchbot merge
